src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -216,7 +216,7 @@ private function getConfiguration()
             return $this->configuration;
         }
         if (false === $mode = $this->mode) {
-            $mode = getenv('SYMFONY_DEPRECATIONS_HELPER');
+            $mode = $this->mode = getenv('SYMFONY_DEPRECATIONS_HELPER');
```

Comment:
are you sure? isn't it memoization taken care of by the "configuration" property now?

Reply:
Oh you're correct! I'll update the PR to just remove that property then.